### PR TITLE
Add "Updating Your Tracker's Firmware" page

### PR DESCRIPTION
**Pull request by kounocom**

### CODE REVIEW

File: src/updating-firmware.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+![Tracker R11](assets/img/firmware_flashpin_r11.jpg)
+![Tracker R12](assets/img/firmware_flashpin_r12.jpg)
+
+5. While still doing step 4, turn on your tracker and plug it in to your computer with a USB-C cable. If you did step 4 correctly, the tracker *should not* connect to Wi-Fi and the SlimeVR Server.
+
+6. Open the [SlimeVR Firmware Tool](https://slimevr-firmware.bscotch.ca/) in a Chromium based browser and select the same options described in the OTA method:
+
+![Firmware Options](assets/img/firmware_options.png)
+
+7. Press Flash to Device, and select your tracker in the pop-up, which should appear as `USB Serial`.
```
Comment (TheDevMinerTV) [inline src/updating-firmware.md:80]: The tracker has to be off and connected, it'll show up in the Serial dropdown anyways.
When the flashing is waiting, the tweezers need to pressed on the pads and the shielding, while turning on the tracker, after which the web flasher should continue.

File: src/updating-firmware.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Updating Your Tracker's Firmware
+
+There are two ways to update the firmware on Official SlimeVR trackers: OTA (over-the-air) or via USB.
+
+The currently recommended version of firmware for Official trackers is v0.4.0.
```
Comment (TheDevMinerTV) [inline src/updating-firmware.md:5]:
```suggestion
There are two ways to update the firmware on official SlimeVR trackers: OTA (over-the-air) or via USB.

The currently recommended version of firmware for official trackers is v0.4.0.
```

File: src/updating-firmware.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## Method 1: Updating Over-the-Air
+
+This method is the simplest, and doesn't require any cables or disassembly, but it does require your tracker to already be paired to your Wi-Fi network.
```
Comment (TheDevMinerTV) [inline src/updating-firmware.md:9]:
```suggestion
This method is the simplest, and doesn't require any cables or disassembly, but it does require your tracker to already be paired to your Wi-Fi network and has the potential to mess up your tracker. If it does, please refer to the USB flashing section below.
```

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

**Review by Eirenliel — APPROVED** [review]
Good start but we need more warnings and clarity in some of this. I think it's okay to merge as is rn.

**Pull request merged**